File: packages/java/src/endpoints/scan_schedules.java
package net.tavoai.sdk;

import okhttp3.*;
import com.google.gson.Gson;
import java.util.concurrent.CompletableFuture;
import java.util.Map;
import java.util.HashMap;

/**
 * Client for scan_schedules endpoints
 */
public class ScanSchedulesClient {
    private final OkHttpClient httpClient;
    private final String baseUrl;
    private final Gson gson = new Gson();

    public ScanSchedulesClient(OkHttpClient httpClient, String baseUrl) {
        this.httpClient = httpClient;
        this.baseUrl = baseUrl;
    }

{methods_str}
}}

File: packages/rust/src/endpoints/scan_schedules.rs
//! Scan_Schedules API Client

use reqwest::Client;
use serde_json::Value;
use std::sync::Arc;

pub struct ScanSchedulesClient {
    client: Arc<Client>,
    base_url: String,
}

impl ScanSchedulesClient {
    pub fn new(client: Arc<Client>) -> Self {
        Self {
            client,
            base_url: "https://api.tavo.ai/api/v1".to_string(),
        }
    }

{methods_str}
}}
